docs/development/core/server/kibana-plugin-core-server.pluginmanifest.md

```diff
@@ -26,7 +26,7 @@ Should never be used in code outside of Core but is exported for documentation p
 |  [id](./kibana-plugin-core-server.pluginmanifest.id.md) | <code>PluginName</code> | Identifier of the plugin. Must be a string in camelCase. Part of a plugin public contract. Other plugins leverage it to access plugin API, navigate to the plugin, etc. |
 |  [kibanaVersion](./kibana-plugin-core-server.pluginmanifest.kibanaversion.md) | <code>string</code> | The version of Kibana the plugin is compatible with, defaults to "version". |
 |  [optionalPlugins](./kibana-plugin-core-server.pluginmanifest.optionalplugins.md) | <code>readonly PluginName[]</code> | An optional list of the other plugins that if installed and enabled \*\*may be\*\* leveraged by this plugin for some additional functionality but otherwise are not required for this plugin to work properly. |
-|  [owner](./kibana-plugin-core-server.pluginmanifest.owner.md) | <code>{</code><br/><code>        readonly name: string;</code><br/><code>        readonly githubTeam?: string;</code><br/><code>    }</code> | TODO: make required once all internal plugins have this specified. |
+|  [owner](./kibana-plugin-core-server.pluginmanifest.owner.md) | <code>{</code><br/><code>        readonly name: string;</code><br/><code>        readonly githubTeam?: string;</code><br/><code>    }</code> |  |
```

Comment:
```suggestion
|  [owner](./kibana-plugin-core-server.pluginmanifest.owner.md) | <code>{</code><br/><code>        readonly name: string;</code><br/><code>        readonly githubTeam?: string;</code><br/><code>    }</code> | The team and github team name who can be contacted about the plugin. |
```

Reply:
I don't know why the comment I added didn't get picked up here, but this is an autogenerated file. Here is my comment:

``` 
  /**
     * All internal plugins should have a github team specified. GitHub teams can be viewed here:
     * https://github.com/orgs/elastic/teams
     */
    readonly githubTeam?: string;
```
    
These docs are legacy and will be going away as soon as we can switch to the new system, so if you don't mind, I am just going to leave this alone.